allure-notifications/build.gradle

```diff
@@ -0,0 +1,26 @@
+dependencies {
+    annotationProcessor(group: 'org.projectlombok', name: 'lombok', version: lombokVersion)
+    compileOnly(group: 'org.projectlombok', name: 'lombok', version: lombokVersion)
+
+    implementation project(':allure-notifications-api')
+
+    implementation platform(group: 'org.apache.logging.log4j', name: 'log4j-bom', version: '2.23.1')
+    implementation(group: 'org.apache.logging.log4j', name: 'log4j-api')
+    implementation(group: 'org.apache.logging.log4j', name: 'log4j-core')
+    implementation(group: 'org.apache.logging.log4j', name: 'log4j-slf4j2-impl')
+
+    implementation(group: 'com.konghq', name: 'unirest-java', version: unirestVersion)
```

Comment:
```suggestion
```
it won't be needed if it's marked as `api` in api layer

Reply:
> it won't be needed if it's marked as `api` in api layer

Not sure why, but this dependence is not pulled up transitively (even through `api`)